harvester_e2e_tests/integration/test_hosts.py

```diff
@@ -67,3 +127,156 @@ def test_host_poweron_state(api_client, host_state, wait_timeout):
 
     _, node = api_client.hosts.get(node['id'])
     assert "active" == node["metadata"]["state"]["name"]
+
+
+@pytest.mark.hosts
+@pytest.mark.p0
+def test_verify_host_info(api_client):
+    status_code, nodes_info = api_client.hosts.get()
+
+    assert status_code == 200, f"Failed to list nodes with error: {nodes_info}"
+    assert len(nodes_info['data']) >= 1, "Incorrect hosts amount"
+
+    for node in nodes_info.get('data', []):
+        node_name = node.get('metadata', {}).get('name', "")
+        assert "" != node_name, ("Node name can't be empty")
+
+        cpus = node.get('status', {}).get('capacity', {}).get('cpu', 0)
+        assert 0 != cpus, (
+            f"Node {node_name} cpu should not be zero"
+        )
+
+        mems = node.get('status', {}).get('capacity', {}).get('memory', "0Ki")
+        mems = "".join(c for c in mems if c.isdigit())
+        mems = mems or "0"
+        assert 0 != int(mems, 10), (
+            f"Node {node_name} memory should not be zero"
+        )
+
+
+@pytest.mark.hosts
+@pytest.mark.p0
+class TestHostTriggerVmMigrate:
+    @pytest.mark.dependency()
+    def test_prerequisite(self, api_client):
+        status_code, nodes_info = api_client.hosts.get()
+
+        assert status_code == 200, f"Failed to list nodes with error: {nodes_info}"
+        assert len(nodes_info['data']) > 1, "Please use multiple hosts to test vm migration"
+
+        for node_info in nodes_info.get('data', []):
+            for condition in node_info.get('status', {}).get('conditions', []):
+                if condition.get('type', "") == "Ready":
+                    assert "True" == condition.get('status', ""), (
+                        f"Node {node_info['metadata']['name']} is not ready"
+                    )
+            assert False is node_info.get('spec', {}).get('unschedulable', False), (
+                f"Node {node_info['metadata']['name']} is unschedulable"
+            )
+
+    @pytest.mark.dependency(depends=["TestHostTriggerVmMigrate::test_prerequisite"])
+    def test_maintenance_mode(self, api_client, focal_vm, wait_timeout):
+        vm_name = focal_vm['metadata']['name']
+        vm_namespace = focal_vm['metadata']['namespace']
+        code, data = api_client.vmis.get(name=vm_name, namespace=vm_namespace)
```

Comment:
If we will just need VM's name and namespace, we could update fixture to return them directly, then we can make the code looks like:
```python
code, data = api_client.vms.get_status(focal_vm['name'], focal_vm['namespace'])
```
or in further:
```python
code, data = api_client.vms.get_status(**focal_vm)
```